css/src/components/sidebar.css

```diff
@@ -10,6 +10,7 @@
 
 .yoast-main {
 	margin-right: 24px;
+	flex: 1;
```

Comment:
I suggest changing this to flex-grow: 1; that's better responsive for smaller screens